### ISSUE TITLE
Maximum length of input

### ISSUE THREAD

**Issue by Stormur** [chartes/deucalion-model-lasla#5]
Is there a maximum size for the input? When trying to tag a text of ca. 35000 tokens, I obtain an analysis only for the first 12000-13000 or so. 

Thank you!

**Comment by PonteIneptique** [comment]
I don't think it should have stopped. The response is streamed normally. Pardon me asking, but did you wait for the stream to end ? I just want to eliminate this possibility

**Comment by Stormur** [comment]
Yes, of course, it is a legitimate question! Yes, I waited, then at some point it stopped. I will try again, however.

**Comment by PonteIneptique** [comment]
It could be a length issue. I have planned to provide some tuning capacities (if you send a lot of long sentence, given RAM and CPU, it cant crash the service) in the near future. I'll be AFK for 10 days though, so hopefully my help was enough ATM :)

**Comment by PonteIneptique** [comment]
Dear @Stormur , there is indeed an issue with the tool right now, as the streaming that happens is broken the Gunicorn server. I'll have to look into that later ;)

**Comment by PonteIneptique** [comment]
Note that a fix is coming for this. There were two different issues : 
- Batch Size can lead to breaking the service
- NGinx would stop streaming the content after some time (30/45 sec). It's good now.
